app/resonant-laboratory/grunt-watch-plugin.js

```diff
@@ -12,7 +12,7 @@ module.exports = {
   apply: function (compiler) {
     compiler.plugin('emit', (compilation, callback) => {
       if (compilation.options.watch === true) {
-        exec('cd $GIRDER_PATH && grunt', function (error, stdout, stderr) {
+        exec('cd $GIRDER_PATH && grunt --force', function (error, stdout, stderr) {
```

Comment:
For some reason grunt is tripping up (and crashing) on the `database_assetstore` plugin. It's possible there's something weird in my girder installation as a result of switching branches... I'll try a fresh girder installation and see if this problem goes away.